runtime/karura/src/xcm_config.rs

```diff
@@ -161,6 +157,14 @@ parameter_types! {
 		// BNC:KSM = 80:1
 		ksm_per_second() * 80
 	);
+	pub BncPerSecondOfCanonicalLocation: (AssetId, u128) = (
+		MultiLocation::new(
+			0,
+			X1(GeneralKey(parachains::bifrost::BNC_KEY.to_vec())),
```

Comment:
BNC shouldn't be our side token, for the testcase working, use register native asset.

Reply:
try register foreign asset, as foreign asset has Trader while native asset do not exist Trader.